quantum/quantum_keycodes.h

```diff
@@ -425,6 +425,7 @@ enum quantum_keycodes {
     RGB_MODE_GRADIENT,
     RGB_MODE_RGBTEST,
 
+#ifdef SPACE_CADET_ENABLE
```

Comment:
Please remove these. We want the keycodes to be defined even if they don't do anything.